Frontend/Frontend.Client/Components/MessageComponent.razor

```diff
@@ -49,7 +50,6 @@
     }
 
     .user {
```

Comment:
is this user selector now removable since it's empty?

Reply:
Yeah thought it might be needed but I can get rid of it 👍